src/fastcs/util.py

```diff
@@ -3,3 +3,42 @@ def snake_to_pascal(input: str) -> str:
     return "".join(
         part.title() if part.islower() else part for part in input.split("_")
     )
+
+def unpack_status_arrays(parameter: list, uri: list[list[str]]):
```

Comment:
```suggestion
def unpack_status_arrays(parameters: list[OdinParameter], uri: list[list[str]]):
```

Reply:
Wouldn't make a list of **OdinParameter** be kind of a problem? Because OdinParameter is a class inside fastcs-odin which depends on fastCS

Reply:
Oh! This is a good point. I think this needs to work in a completely different way, but I think it will be a bit more work. I will make an issue for this.

Let's go back to the original fastcs-odin PR for now, sorry!